app/src/main/java/com/hammer/talkbbokki/data/entity/CommentsItemEntity.kt

```diff
@@ -0,0 +1,5 @@
+package com.hammer.talkbbokki.data.entity
```

Comment:
🚫 **[ktlint]** <standard:filename> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>File 'CommentsItemEntity.kt' contains a single class and possibly also extension functions for that class and should be named same after that class 'CommentsCountEntity.kt'